packages/crayons-core/src/components/toast/toast-util.ts

```diff
@@ -93,6 +98,25 @@ export function createToastNotification(
   containerElem.appendChild(toastElem);
 }
 
+export function preventDuplicates(
+  toasteContainerCollection: HTMLCollection,
+  newToasterOption: ToastOptions
+): boolean {
+  let hasDuplicates = false;
+  Array.from(toasteContainerCollection).forEach((existingToast) => {
+    if (!newToasterOption.shouldPreventDuplicates) return;
```

Comment:
We can skip the forEach altogether if the flag is false. Bring it to the top/wrap the Array.from.forEach in this if statement, meaning only execute the loop if the flag is true

Reply:
yes sandeep, if you see the below file where this function gets invoked we have handled that logic. I think that covers the idea of not executing this function itself when the flag is turned off. wdyt @sandeepbonala-fw?

Reply:
@madhan028 You can remove this `if` condition here. since you are already checking for this condition. so inside the `forEach` loop, `shouldPreventDuplicates` will always be `true` only. You are doing this check in https://github.com/freshworks/crayons/pull/888/files#diff-ae7c2919363874eb1d6cf1f9dbf528ba39ae06d1507cc5c8d5b84dbd61063d0eR64